### DISCUSSION TITLE
Suggestion about the Dot Plot of my sequence

### DISCUSSION THREAD

**Discussion by u7068758** [cogent3/cogent3#1059] [Q&A] [answered]
Hi!
I have performed a Reverse complement DotPlot of a nucleotide sequence (180bp) of a proline-rich region containing tandem repeats. 
(window=20, threshold=10). Can you suggest whether these parameters will work? 

The blue lines indicate that this region has a cluster of repeats and the red lines indicate the matching residues on the reverse complement strand. Would you like to add more to the interpretations?

![image](https://user-images.githubusercontent.com/107029444/174006545-40f0e6b0-4cd9-4090-bd86-6d2def4bb675.png)

**Comment by GavinHuttley** [comment, marked as answer]
This is a duplicate of the another discussion #1057 so I'm locking this.
